src/lib/components/ui/FileInputField/__tests__/FileInputField.test.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import {
+  shallow,
+  mount,
+} from 'enzyme';
+import { shallowToJson } from 'enzyme-to-json';
+import sinon from 'sinon';
+import { FileInputField } from '../FileInputField';
+
+describe('rendering', () => {
+  it('renders correctly mandatory props only', () => {
+    const tree = shallow(<FileInputField id="test" label="label" />);
+
+    expect(shallowToJson(tree)).toMatchSnapshot();
+  });
+
+  it('renders correctly with hidden label', () => {
+    const tree = shallow(<FileInputField
+      id="test"
+      isLabelVisible={false}
+      label="With hidden label"
+    />);
+
+    expect(shallowToJson(tree)).toMatchSnapshot();
+  });
+
+  it('renders correctly with custom props', () => {
+    const tree = shallow(<FileInputField
+      accept=".pdf,.jpg,.jpeg,.png"
+      id="test"
+      label="With custom props"
+      multiple
+    />);
+
+    expect(shallowToJson(tree)).toMatchSnapshot();
+  });
+
+  it('renders correctly with all props', () => {
+    const tree = shallow(<FileInputField
+      disabled
+      fullWidth
+      helpText="some help"
+      id="test"
+      inFormLayout
+      isLabelVisible={false}
+      label="All props"
+      layout="horizontal"
+      required
+      validationState="invalid"
+      validationText="some error"
+      value="value"
+    />);
+
+    expect(shallowToJson(tree)).toMatchSnapshot();
+  });
+});
+
+describe('functionality', () => {
+  it('calls changeHandler()', () => {
+    const spy = sinon.spy();
+    const component = mount(<FileInputField
+      changeHandler={spy}
+      id="test"
+      label="label"
+    />);
+
+    component
+      .find('input')
+      .simulate('change', {
+        preventDefault: () => {},
+        target: { value: 'test' },
```

Comment:
It is called `event.target.files` for `type=file`

Reply:
so it should read: `target: { files: ['test.jpg'] },`